documentation/src/main/asciidoc/upgrading/upgrading.asciidoc

```diff
@@ -48,6 +48,9 @@ Interfaces `DeltaAware`, `Delta` and `CopyableDeltaAware` have been deprecated.
 
 Any partial updates to an entry should be replaced using the link:../user_guide/user_guide.html#functional_map_api[Functional API].
 
+=== Store Changes
```

Comment:
Might as well add a line or 2 to the User Guide as well. Also maybe we should call this `Store Batch Size Changes` or something.